code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1078,8 +1078,12 @@
 	if((H.status_flags & GODMODE))
 		return
 
-	if(!breath || (breath.total_moles() == 0))
-		if(H.reagents.has_reagent("epinephrine"))
+	var/lungs = H.getorganslot("lungs")
+	if(!lungs)
```

Comment:
Yes, but if those species didn't have lungs they'd take the oxy damage from this.
this is my point.